docker/kayobe/Dockerfile

```diff
@@ -36,5 +43,15 @@ WORKDIR /stack
 USER stack
 
 RUN mkdir /stack/.ssh && chmod 700 /stack/.ssh
-COPY --chown=stack:stack ssh_config /stack/.ssh/config
+COPY --chown=stack:stack .automation/docker/kayobe/ssh_config /stack/.ssh/config
```

Comment:
Just got:
![image](https://user-images.githubusercontent.com/19818516/125605424-95ca20fb-3844-4d74-b780-4324ff1e3e6f.png)

Reply:
sorry, this slightly changes the way you have to build it:

```
build kayobe:
  extends:
    - .environment-prepare
    - .tags
  image: docker:stable
  rules:
    - when: always
  services:
    - name: docker:dind
      command: ["--mtu=1450"]
  stage: build
  script:
    - docker login -u $CI_REGISTRY_USER -p $CI_REGISTRY_PASSWORD $CI_REGISTRY
    - docker pull $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA && exit 0 || true
    # Set up ssh-agent to login into private kayobe repo
    # - .automation/utils/kayobe-automation-ssh-agent
    # If we've already built it, save some time and don't build it again
    - DOCKER_BUILDKIT=1 docker build --cache-from $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA -f .automation/docker/kayobe/Dockerfile -t $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA .
    - docker push $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA
```
Namely using BUILDKIT and building from the root of the repository with `-f`